Gemfile

```diff
@@ -1,6 +1,6 @@
 source 'https://rubygems.org'
 
-ruby '2.3.1'
+ruby '2.3.3'
 
-gem 'jekyll'
-gem 'html-proofer'
```

Comment:
I know what it's used for. If they run Bundle update, their local lockfile gets updated and we'll see that in the PR. This doesn't stop that but holds us back from keeping uptodate with these Gems when we do run it.